packages/server/package.json

```diff
@@ -17,11 +20,13 @@
     "axios": "^1.5.0",
     "cors": "^2.8.5",
     "dotenv": "^16.3.1",
+    "enforce-unique": "^1.2.0",
```

Comment:
Just for a single purpose do not install new lib . Come up with your typescript solution.

Reply:
Let it be. It's alright